dnssec.go

```diff
@@ -392,6 +392,12 @@ func (rr *RRSIG) Verify(k *DNSKEY, rrset []RR) error {
 	if k.Protocol != 3 {
 		return ErrKey
 	}
+	// RFC 4034 2.1.1 If bit 7 has value 0, then the DNSKEY record holds some
```

Comment:
We might want to document both this and the k.Protocol in the function doc as well

Reply:
I added a mention in the function doc as well. :+1: